canvas_modules/harness/src/client/App.js

```diff
@@ -1302,13 +1302,13 @@ class App extends React.Component {
 		this.setState({ showPropertiesDialog: true, propertiesInfo: propsInfo });
 	}
 
-	closePropertiesEditorDialog() {
+	closePropertiesEditorDialog(calledFrom) {
```

Comment:
Any reason to change this and the one below?

Reply:
no reason, added for testing but I will remove them